linera-storage/src/db_storage.rs

```diff
@@ -215,6 +215,7 @@ pub struct DbStorage<Store, Clock> {
     wasm_runtime: Option<WasmRuntime>,
     user_contracts: Arc<DashMap<UserApplicationId, UserContractCode>>,
     user_services: Arc<DashMap<UserApplicationId, UserServiceCode>>,
+    blobs: Arc<DashMap<BlobId, Blob>>,
```

Comment:
Are we actually using this anywhere?

Reply:
Ah, not anymore. Good catch!

Reply:
If it's a cache, it would probably be an LRU cache, not a map, otherwise it can grow without bounds.

Which tests are you referring to? Most test have some kind of storage, even if it's just memory?

Reply:
It's analogous to what we already do for example in tests for `user_contracts` and `user_services` https://github.com/linera-io/linera-protocol/blob/fefa995f864cdcb3a101b2354c01077853afa421/linera-execution/src/test_utils/mod.rs#L76
But we also had to do it for blobs so they're accessible during the tests. There could be a way of removing it, but we would have to write straight to storage, and since that's not what we're doing for contract and service code, I thought that wasn't the right approach 🤔 lmk if I'm wrong though